templates/careers/project-management.html

```diff
@@ -1,26 +1,99 @@
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default u-no-margin--bottom">Working here</h2>
+      <h3>Projects you'll work on</h3>
+    </div>
+    <div class="col-6">
+      <p>Project and program managers at Canonical lead the delivery of a wide variety of projects ranging from commercial customer projects, like enablement of Ubuntu on new hardware and the delivery of complex private clouds, to internal Canonical strategic projects and programs.</p>
+    </div>
+  </div>
+  <div class="p-strip is-shallow">
+    <div class="u-fixed-width u-hide--small u-hide--medium">     
+      {{ image (
+        url="https://assets.ubuntu.com/v1/6195ae44-department-project-management-%402x.jpg",
+        alt="",
+        width="2216",
+        height="834",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+  
+  <div class="row">
+    <div class="col-6">
+      <h3>Project Management at Canonical</h3>
+    </div>
+    <div class="col-6">
+      <p>Canonical Project managers spearhead Canonical's Execution Excellence Strategy. They are responsible for ensuring project success through owning, driving, communicating and respectfully engaging all stakeholders, valuing integrity and accountability.Project managers at Canonical, create, manage and maintain project-specific schedules ensuring projects are delivered within time, resource and scope expectations. Each project manager is responsible for multiple projects simultaneously. Managing each project through its life-cycle and ensuring that the goals for both Canonical and the client are met.</p>
+      <p>One advanced role is the Technical Program Manager, who provides technical account leadership and insight as the primary technical interface for our key enterprise accounts. The Technical Program Manager is a high-profile technical engagement program management position where customer relationship management and advocacy for Canonical's technical assets are strategically crucial.</p>
+    </div>
+  <div class="p-strip is-shallow">
+    <hr>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h3>Who you are</h3>
+    </div>
+    <div class="col-6">
+      <p>All Program Managers are responsible for facilitating customer-focused collaboration, team upskilling and driving innovation initiatives. Program managers proactively evaluate enterprise client situations and drive client vision that could ultimately lead to additional revenue-generating business opportunities.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>See for yourself the work we're doing</h2>
+    <ul class="p-inline-list">
+      <li class="p-inline-list__item u-no-margin--right">
+        <a class="github-button" href="https://github.com/canonical" data-size="large" aria-label="Follow @canonical on GitHub"></a>
+      </li>
```

Comment:
Maybe this could use https://vanillaframework.io/docs/patterns/icons#icons-as-links?